### ISSUE TITLE
Adding tests for processing-android

### ISSUE THREAD

**Issue by ranaaditya** [processing/processing-android#644]
**Currently we don't have any tests written for this project as its really hard and complex to write tests for GLSL shaders and OpenGL-ES programs.**

**_But I think we can write tests at least for Utility functions and other basic APIs._**

Resources to learn basics of testing-

[JUNIT5 - testing framework for Java and JVM](https://junit.org/junit5/)
[how to start writing tests for GLSL shaders](https://blogs.igalia.com/siglesias/2015/05/29/piglit-iii-how-to-write-glsl-shader-tests/)
[Writing tests in Android](https://developer.android.com/training/testing)
[Android OpenGL unit testing](https://www.silverbaytech.com/2014/10/03/android-opengl-unit-testing/)

What do you think @codeanticode  about this ?

**Comment by codeanticode** [comment]
Yes, I think we should have some testing infrastructure for the android mode. This is an area I'm not very experienced though, perhaps you can lead this effort.

**Comment by ranaaditya** [comment]
> Yes, I think we should have some testing infrastructure for the android mode. This is an area I'm not very experienced though, perhaps you can lead this effort.

yes @codeanticode , I was figuring out how can we start writing unit tests for this project and how to design the testing infrastructure so that new contributors can easily catchup our progress.

yeah definitely I will lead the testing phase for android-mode.

Best,
Aditya
